### DISCUSSION TITLE
Ailerons in recordings

### DISCUSSION THREAD

**Discussion by TwattyBollox** [nguyenquyhy/Flight-Recorder#88] [General]
Hi

Is it a problem with the recorder, MSFS, or FBW A32NX that is causing the ailerons to go the wrong way?

It's a little strange to see them in recorded videos going up and the wing also going up :¬)

Thanks for the great work so far  ;¬)

**Comment by nguyenquyhy** [comment]
Yep I mentioned this in the description in flightsim.to (https://flightsim.to/file/8163/flight-recorder/129552 Incompatibility section). Probably I should update the README here too :joy:.

**Reply by TwattyBollox** [reply]
Sry ... My mistake, I did not see that one :¬)

**Comment by nguyenquyhy** [comment]
A fix for this is now released in v0.24. Please try it out if you have a chance. The fix works in old recording too.
